ci/docker/runtime_functions.sh

```diff
@@ -820,6 +843,7 @@ build_ubuntu_gpu_cmake_no_rtc() {
         -DUSE_SIGNAL_HANDLER=ON                 \
         -DUSE_CUDA=ON                           \
         -DUSE_CUDNN=ON                          \
+        -DUSE_TVM_OP=OFF                         \
```

Comment:
how does it fail with TVM_OP=OFF? http://jenkins.mxnet-ci.amazon-ml.com/blue/rest/organizations/jenkins/pipelines/mxnet-validation/pipelines/unix-gpu/branches/PR-18678/runs/3/nodes/59/steps/293/log/?start=0

Reply:
Thanks! It used to be on when producing that log.